code/code/cmd/cmd_deathstroke.cc

```diff
@@ -98,19 +98,20 @@ static int deathstroke(TBeing *caster, TBeing *victim)
       if (caster->reconcileDamage(victim, 0,sktype) == -1)
         return DELETE_VICT;
     }
-    /* with great failure, comes great sorrow *grin* */
-    percent = ((10 - (caster->getArmor() / 50)) << 1);
-    percent += caster->getDexReaction() * 5;
-    percent -= victim->getAgiReaction() * 5;
+
+    // with great failure, comes great sorrow *grin*
     // what happens here is that the mob gets a shot at the players vitals
     // ... fair is fair right? 
+    percent = ((10 - (caster->getArmor() / 50)) << 1);
+    percent += victim->getDexReaction() * 5;
+    percent -= caster->getAgiReaction() * 5;
```

Comment:
Why'd you flip them? Dex is offense, Agi is defense, so it makes sense to check victim's Agi when attacking.

Reply:
this calculation is for the counterattack, where the "victim" is the attacker. the previous code was a copy-paste error.

Reply:
Ohhh right!